components/dashboard/src/app/AppRoutes.tsx

```diff
@@ -125,24 +124,10 @@ export const AppRoutes = () => {
                             <Route path="/open">
                                 <Redirect to="/new" />
                             </Route>
-                            {/* TODO(gpl): Remove once we don't need the redirect anymore */}
-                            <Route
-                                path={[
-                                    switchToPAYGPathMain,
-                                    settingsPathPlans,
-                                    "/old-team-plans",
-                                    "/teams",
-                                    "/subscription",
-                                    "/upgrade-subscription",
-                                    "/plans",
-                                ]}
-                                exact
-                            >
-                                <Redirect to={"/billing"} />
-                            </Route>
```

Comment:
I'll probably add this back, was playing around the router, and there is most likely no harm in keeping this.